src/components/Button/index.test.js

```diff
@@ -43,13 +43,15 @@ describe('Button component', () => {
         disabled	
         isFullWidth	
         isBreakoutLink
+        margin="all-s"
```

Comment:
Shouldn't this error with only a single hyphen?

Reply:
yup you're right :)